### DISCUSSION TITLE
Laptops to Run PHoeNIx

### DISCUSSION THREAD

**Discussion by Siddra2021** [CDCgov/phoenix#110] [Q&A]
Hello, 
I will be purchasing a new laptop for work. Would you recommend any specifications. I will be using it to run PHoeNIx pipeline. 

Siddra

**Comment by jvhagey** [comment]
Hi @Siddra2021, we have a high powered laptop that will run it (12CPU/2 threads per, 2.8Ghz, 64GBs RAM). This was purchased several years ago so you can probably get something a bit better by this point.

**Comment by jvhagey** [comment]
These are the specs on a MacBook that can run PHoeNIx.

![image](https://github.com/CDCgov/phoenix/assets/21023514/bcfcba32-5785-4586-80a9-19242a050401)
